bedrock/events/tests/test_data/reps.ical

```diff
@@ -10,7 +10,7 @@ DTSTART;TZID=Europe/Skopje:20141007T180000
 DTEND;TZID=Europe/Skopje:20141007T190000
 LOCATION:KIKA\, Skopje\, Macedonia\, F.Y.R. of
 SUMMARY:Community meeting in Skopje
-DESCRIPTION:It&#39;s been a long summer and finally we have enough people back
+DESCRIPTION:It&#39;s been a long summer and finally we have enough people back
```

Comment:
I know this is not your change, but I just noticed it randomly: iCalendar doesn't use html escapes. This should be `'` and if html escaping is needed then it should probably be done elsewhere